benchmarks/callbacks/benchmark_SequenceOfRegionTimer.cpp

```diff
@@ -0,0 +1,149 @@
+#include <iostream>
+
+#include "benchmark/benchmark.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/callbacks/EventNameMatcher.hpp"
+#include "kokkos-utils/callbacks/Manager.hpp"
+#include "kokkos-utils/callbacks/SequenceOfRegionTimer.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+namespace Kokkos::utils::benchmarks::callbacks
+{
+struct MyFixture
+{
+    void run(const ::benchmark::State& state) const
```

Comment:
We could keep `MyWorkload` and `execute` as names for them.

Reply:
:heavy_check_mark: